Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,92 @@
+# Preparing development environment and first rkt build
+This is an example configuration for a fresh installation on Ubuntu 16.04 GNOME.
+This doc is also a short quick start guide.
+For a detailed reference see [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Get rkt repo and install dependencies
+In this example ~/Repos is a personal workspace where all repos are stored
```

Comment:
`Repos` is just a bunch of symlinks into a dot-hidden GOPATH. Couldn't this document be shorter -- by cloning rkt directly into a real directory in a "standard" GOPATH? Like `~/go/src/github.com/coreos/rkt`? Skip all the workspace `mkdirs` and `ln`s?